index.html

```diff
@@ -12,10 +12,97 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
+<!--navigation bar -->
```

Comment:
You can format the code in this file to make to easier to read by using the instructions here
https://stackoverflow.com/a/29973358